app/messaging/message-request-schema.js

```diff
@@ -0,0 +1,20 @@
+const joi = require('joi')
+
+const applicationMessageSchema = joi.object({
+  crn: joi.string().max(14).required(),
+  sbi: joi.string().max(14)
+})
+
+const validateMessageRequest = (logger, event) => {
+  const validate = applicationMessageSchema.validate(event)
+  if (validate.error) {
+    logger.error('Message request validation error', JSON.stringify(validate.error))
```

Comment:
Another general point. Better to log the error as an object I think, Pino is heavily optimised for this and will catch errors during attempts to stringify. Also means we can consistently rely upon serialisers and redaction.

```
logger.error({  err }, 'message')
```

Reply:
Will update and commit once you've completed round one of code review. Thanks